cfme/v2v/migrations.py

```diff
@@ -132,12 +138,11 @@ def fill(self, values):
             self.target_clusters.fill(mapping['target'])
             self.add_mapping.click()
         was_change = not self.mappings_tree.is_empty
-        self.next_btn.click()
+        self.after_fill()
```

Comment:
I don't think this is quite right here. `after_fill` is called by `View.fill()`

https://github.com/RedHatQE/widgetastic.core/blob/master/src/widgetastic/widget/base.py#L987

Reply:
@jawatts : I also thought so .. but something is different in the fill here. 
The after_fill was not being called automatically .
May be  @mshriver will be able to tell why it is not called .

Reply:
This method is overriding View.fill(), and yes needs to call after_fill() just like View.fill() does.  The call needs to pass was_change, and the overriding implementation should define after_fill to take the value, even if its not used, for consistency.